backend/src/services/promptService.ts

```diff
@@ -0,0 +1,3 @@
+export const create = (theme: string, input: string): string => {
+    return theme + input;
```

Comment:
Appending of strings should contain space between theme and input

Reply:
Good point! I'll go fix this :)